### PR TITLE
chore: warn about mis-configured resources in debug:api-resource command

### PR DESCRIPTION
**Pull request by azjezz**
Signed-off-by: azjezz <azjezz@protonmail.com>

| Q             | A
| ------------- | ---
| Branch?       | main
| Tickets       | N/A
| License       | MIT
| Doc PR        | N/A

### CODE REVIEW

File: src/Symfony/Bundle/Command/DebugResourceCommand.php [hunk 1]
```diff
@@ -24,8 +25,12 @@
 
 final class DebugResourceCommand extends Command
 {
-    public function __construct(private readonly ResourceMetadataCollectionFactoryInterface $resourceMetadataCollectionFactory, private readonly ClonerInterface $cloner, private $dumper)
-    {
+    public function __construct(
```
Comment (alanpoulain) [inline src/Symfony/Bundle/Command/DebugResourceCommand.php:28]: We follow the style of Symfony: everything on the same line.

**Comment by soyuka** [comment]
I don't get it, the resource metadata factory is already calling the extractor ?

**Comment by azjezz** [comment]
depends on which resource metadata factory is used, attribute resource metadata factory doesn't call the extractor, and will create the resource even if it's not configured ( related to the internal discussion )

**Comment by soyuka** [comment]
We use decoration on our `api_platform.metadata.resource.metadata_collection_factory` service and it should definitely call https://github.com/api-platform/core/blob/main/src/Metadata/Resource/Factory/ExtractorResourceMetadataCollectionFactory.php at some point.

**Comment by azjezz** [comment]
> We use decoration on our api_platform.metadata.resource.metadata_collection_factory service and it should definitely call https://github.com/api-platform/core/blob/main/src/Metadata/Resource/Factory/ExtractorResourceMetadataCollectionFactory.php at some point.

Yes, but the attribute resource metadata collection factory result won't be empty, hence why the command doesn't fail, or emit a warning.

**Pull request closed without merging**